dattri/func/ihvp.py

```diff
@@ -784,16 +785,23 @@ def ihvp_datainf(
         `v`(layer-wise) and returns the approximated IHVP of the approximated Hessian of
         `func` and `v`.
     """
+    # TODO: param_layer_map should not be optional.
```

Comment:
The reason why `param_layer_map` will always have input if users use `Attributor`. But it should not be `None` if users use the `ihvp_datainf` directly (currently there is no guarantee). So I leave a TODO here since it's a minor problem.